### DISCUSSION TITLE
For Project V2, updating single select options on items could be improved

### DISCUSSION THREAD

**Discussion by timlevett** [community/community#52558] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Hi all, big fan of the new Projects. I'm working on building a github action that updates item fields and I ran into a challenge. I think its super easy to update a text or date field, but updating a single select option is rather challenging. You have to find the option id, then use the option id in the update mutation. It would be nice if you could pass the option name instead, it would save an API call. Idea on how you could do this, try to find a match, and if you don't have one, then create a new option.

**Comment by G-Ke** [comment]
I too, struggled with updating the Status field of a Project Item. I had to go into the UI, open DevTools, and find the Option ID by manually changing the Status. If there is no other way to get those Option IDs for the --single-select-option-id argument, passing the name would be a much better experience. 

Edit: This was my experience when using the gh cli. Let me know if you have any questions!
